packages/cli/src/commands/doctor/healthchecks/androidSDK.ts

```diff
@@ -61,13 +82,88 @@ export default {
       needsToBeFixed: !isRequiredVersionInstalled,
     };
   },
-  runAutomaticFix: async ({loader, environmentInfo}) => {
-    const version = environmentInfo.SDKs['Android SDK'];
-    const isSDKInstalled = version !== 'Not Found';
+  win32AutomaticFix: async ({loader}) => {
+    // TODO: Create a GitHub action that checks and updates this periodically?
+    const cliToolsUrl =
+      'https://dl.google.com/android/repository/commandlinetools-win-6200805_latest.zip';
+
+    const systemImage = 'system-images;android-28;google_apis;x86_64';
+    // Installing 29 as well so Android Studio does not complain on first boot
+    const componentsToInstall = [
+      'platform-tools',
+      'build-tools;29.0.3',
+      'platforms;android-29',
+      // Is 28 still needed?
+      'build-tools;28.0.3',
```

Comment:
This should probably be tracked too, if it means we may end up downloading and installing tooling our users don’t need?

Reply:
Do you have a link to the discord chat?

Reply:
https://discord.com/channels/514829729862516747/514832058955202561/694436642454372362